generation/WinSDK/enums.json

```diff
@@ -34707,6 +34707,143 @@
       }
     ]
   },
+  {
+    "type": "uint",
+    "name": "DXGI_PRESENT",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "DXGI_PRESENT_",
+      "header": "dxgi.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "interface": "IDXGISwapChain",
+        "method": "Present",
+        "parameter": "Flags"
+      },
+      {
+        "interface": "IDXGISwapChain1",
+        "method": "Present1",
+        "parameter": "PresentFlags"
+      },
+      {
+        "interface": "IDXGIDecodeSwapChain",
+        "method": "PresentBuffer",
+        "parameter": "Flags"
+      }
+    ]
+  },
+  {
+    "type": "uint",
+    "name": "DXGI_ENUM_MODES",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "DXGI_ENUM_MODES_",
+      "header": "dxgi.h|dxgi1_2.h"
```

Comment:
I need some help with this enum. Two constants are defined in `dxgi.h` and two more in `dxgi1_2.h`.

I tried playing around with `addUsesTo`, but as the name implies that is only for `uses` and not for `autoPopulate`'ing extra constants.

Reply:
@mikebattista can I get some help here?

Reply:
The tooling doesn't support multiple headers; you'll have to define them manually for now.

Reply:
It seems very trivial to update the tooling to support a `|` or `;` separator.